website/docs/guides/1.8.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-180-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.8.0 has a number of new and exciting features and changes.
+
+**Major Enhancements:**
+* The HashiCorp Terraform MongoDB Atlas Provider now supports [Snapshot Distribution](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_backup_schedule) 
+  * To learn more about this feature see here: https://www.mongodb.com/blog/post/introducing-snapshot-distribution-atlas   
```

Comment:
We normally start these with a "You can now..." as that was feedback from docs when they were able to review.  It makes it a bit more personal.  Also you can work in the docs link for the feature into the context to make it a bit less cluttered.  Providing example to help illustrate
```suggestion
* You can now enable and manage backup [snapshot distribution](https://www.mongodb.com/blog/post/introducing-snapshot-distribution-atlas) with [`mongodbatlas_cloud_backup_schedule`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_backup_schedule)
```

Reply:
updated